lambdas/abandon/src/main/java/uk/gov/di/ipv/cri/kbv/api/handler/AbandonKbvHandler.java

```diff
@@ -4,20 +4,69 @@
 import com.amazonaws.services.lambda.runtime.RequestHandler;
 import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyRequestEvent;
 import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyResponseEvent;
+import software.amazon.awssdk.awscore.exception.AwsServiceException;
 import software.amazon.awssdk.http.HttpStatusCode;
+import uk.gov.di.ipv.cri.common.library.service.ConfigurationService;
+import uk.gov.di.ipv.cri.common.library.service.SessionService;
+import uk.gov.di.ipv.cri.common.library.util.EventProbe;
+import uk.gov.di.ipv.cri.kbv.api.service.KBVStorageService;
 
 import java.util.Map;
+import java.util.UUID;
+
+import static org.apache.logging.log4j.Level.ERROR;
 
 public class AbandonKbvHandler
         implements RequestHandler<APIGatewayProxyRequestEvent, APIGatewayProxyResponseEvent> {
+    public static final String ABANDON_STATUS = "Abandoned";
```

Comment:
these static variables can be made private or simply removed (HEADER_SESSION_ID / ABANDON_STATUS )  if the string is only used in one place.
Broadly speaking, we shouldn't make variables public just for unit tests. 

For example, if the unit test didn't exist would the variables have been declared with the private access modifier?
In the unit test it is acceptable to simply duplicate the strings.

Reply:
It could also be argued that a common-lib RequestUtils (or similar) class could handle the retrieval of the session id from the request